massa-network-exports/src/settings.rs

```diff
@@ -118,6 +122,8 @@ pub mod tests {
                 max_in_connection_overflow: 2,
                 peer_types_config,
                 max_operations_per_message: MAX_OPERATIONS_PER_MESSAGE,
+                max_bit_read: std::u64::MAX,
```

Comment:
except the name which should be max_bytes_per_second_r or similar, what is the difference between this setting and the one above (NetworkSettings::max_bit_read) ?